app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
```

Comment:
Add tests for negative numbers and incorrect types

Reply:
Add one more test case if the function receives an incorrect type of data it raises the correct exception.